packages/components/app/styles/components/dropdown.scss

```diff
@@ -208,18 +209,23 @@ $hds-dropdown-toggle-border-radius: 5px;
   color: var(--token-color-foreground-action);
 }
 
+// HDS::DROPDOWN::LIST-ITEM::DESCRIPTION
+
 .hds-dropdown-list-item--description {
   padding: 2px 16px 4px;
   color: var(--token-color-foreground-faint);
 }
 
+// HDS::DROPDOWN::LIST-ITEM::GENERIC
+
 .hds-dropdown-list-item--generic {
   padding-right: 16px;
   padding-left: 16px;
 }
 
-
-.hds-dropdown-list-item--interactive {
+// HDS::DROPDOWN::LIST-ITEM::INTERACTIVE & HDS::DROPDOWN::LIST-ITEM::CHECKMARK
+.hds-dropdown-list-item--interactive,
+.hds-dropdown-list-item--checkmark {
```

Comment:
we still need a `--checkmark` modifier operating some overrides, so instead of composing from two modifiers (`--interactive` and `--checkmark` I thought I may use the same properties block. Some of these properties could potentially be abstracted in a mixin to simplify the code a bit.

Reply:
👍